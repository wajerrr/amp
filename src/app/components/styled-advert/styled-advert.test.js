import React from 'react';
import renderer from 'react-test-renderer';
import StyledInlineAd from './styled-advert';

describe('InlineAd', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<StyledInlineAd siteCode="1234" zoneCode="ABCD" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
