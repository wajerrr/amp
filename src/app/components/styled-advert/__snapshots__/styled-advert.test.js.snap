// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineAd should match snapshot 1`] = `
<div
  className="css-idu2pl-StyledInlineAd exmsx7n0"
>
  <amp-ad
    className="css-idu2pl-StyledInlineAd exmsx7n0"
    data-slot="/5605/dev.1234/ABCD"
    height="250"
    type="doubleclick"
    width="300"
  />
</div>
`;
